### PR TITLE
 2021 03 23 vasilemaftita tuesday coding

### PR DESCRIPTION
**Pull request by vasilemaftita**

### CODE REVIEW

File: web/vasilemaftita/html/color-hex-five/colorhexfive.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+h1 {
+    background-color: pink;
+    color: blue;
+    padding: 15px;
+  }
+  h2 {
+    color: brown;
+    padding: 10px;
+  }
```
Comment (robertoalx) [inline web/vasilemaftita/html/color-hex-five/colorhexfive.css:9]: Empty line please

File: web/vasilemaftita/html/color-hsl-five/colorhslfive.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+h1 {
+    background-color: crimson;
+}
+<style>
+p.round1 {
+    border: 2px solid red;
+    border-radius: 5px;
+}
+</style>
```
Comment (robertoalx) [inline web/vasilemaftita/html/color-hsl-five/colorhslfive.css:9]: Empty line here as well

File: web/vasilemaftita/html/css-five/css.css [hunk 3]
```diff
@@ -0,0 +1,8 @@
+h1 {
+    background-color: blueviolet;
+    color: brown;
+    padding: 15px;
+}
+p {
+    color: darkblue;
+}
```
Comment (robertoalx) [inline web/vasilemaftita/html/css-five/css.css:8]: Empty line :)

File: web/vasilemaftita/html/editor-five/editorfive.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+h1 {
+    background-color: brown;
+    color: chartreuse;
+}
+h2 {
+    font-family: Arial, Helvetica, sans-serif;
+}
+p {
+    background-color: silver !important;
+}
```
Comment (robertoalx) [inline web/vasilemaftita/html/editor-five/editorfive.css:10]: Empty line

File: web/vasilemaftita/html/heading-five/headingfive.css [hunk 5]
```diff
@@ -0,0 +1,9 @@
+h1 {
+    background-color: blue;
+    color: burlywood;
+}
+h2 {
+    color: brown;
+    padding: 10px;
+}
+p {color: darkmagenta;}
```
Comment (robertoalx) [inline web/vasilemaftita/html/heading-five/headingfive.css:9]: Empty line :)

**Review by Constantin-Matei — APPROVED** [review]
Nice job EL & !important :)

**Review by robertoalx — APPROVED** [review]
Nice one :)

File: web/vasilemaftita/html/color-hsl-five/colorhslfive.css (hunk 2 above)
Comment (RadosavAntonio) [reply web/vasilemaftita/html/color-hsl-five/colorhslfive.css:9]: you can insert an empty line.

File: web/vasilemaftita/html/css-five/css.css (hunk 3 above)
Comment (RadosavAntonio) [reply web/vasilemaftita/html/css-five/css.css:8]: You can insert an empty line.

File: web/vasilemaftita/html/editor-five/editorfive.css (hunk 4 above)
Comment (RadosavAntonio) [reply web/vasilemaftita/html/editor-five/editorfive.css:10]: You can insert an empty line.

File: web/vasilemaftita/html/heading-five/headingfive.css (hunk 5 above)
Comment (RadosavAntonio) [reply web/vasilemaftita/html/heading-five/headingfive.css:9]: You can insert an empty line.

File: web/vasilemaftita/html/color-hex-five/colorhexfive.css (hunk 1 above)
Comment (RadosavAntonio) [reply web/vasilemaftita/html/color-hex-five/colorhexfive.css:9]: You can insert an empty line.

**Comment by RadosavAntonio** [comment]
do not forget to put empty lines to be easyer to read

**Review by RadosavAntonio — APPROVED** [review]
You can insert an empty line.

**Pull request merged**